fcli-other/fcli-functional-test/src/ftest/groovy/com/fortify/cli/ftest/fod/_common/FODAppRel.groovy

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ */
+package com.fortify.cli.ftest.fod._common
+
+import com.fortify.cli.ftest._common.Fcli
+
+public class FODAppRel implements Closeable, AutoCloseable {
```

Comment:
because creating an app automatically also creates a release

Reply:
I guess there's something to say for both. This class operates on the `fcli fod app` entity, and the corresponding command classes are named `FoDApp*Command`, so in that regards it should be named `FoDApp`. Indeed creating an app also automatically creates a release, and as such this class can also be used for testing release-related commands, so in that regards, something like `FoDAppRel` also makes sense. 

However, `FoDAppRel` maybe interpreted as both 'application with release', or 'application release'. This class represents the first, but most people may expect the latter, especially since previously `fcli fod app-rel` was an alias for `fcli fod release`, and the command classes were named `FoDAppRel*Command`. So, when seeing the `FoDAppRel` class name, people may expect that this class represents 'application releases', rather than 'application with implicit release'.

So, if you'd want the class name to represent the concept of 'application with release', I'd suggest using a more descriptive name.